styleguide/_themes/global/scss/global.scss

```diff
@@ -5,3 +5,4 @@
 @import '_fonts';
 @import '_forms';
 @import '_buttons';
+@import '_bootstrap_overrides';
```

Comment:
I am in favor of a PR that removes the unnecessary underscores on these and other files. In Harp underscores are meant to be excluded but the `_themes` directory excludes everything within it, and the exclusions do not influence Sass imports. We could avoid the single/double quote thing altogether without underscores.